analysis/freq.go

```diff
@@ -9,6 +9,10 @@
 
 package analysis
 
+// TokenLocation represents one occurrence of a term at a particular location in
+// a field. Start, End and Position have the same meaning than in analysis.Token.
```

Comment:
nitpick: "same meaning as..." not "than"

The rest of your changes look good to me! Not sure if you should change your PR because of that nitpick.